ACHNBrowserUI/ACHNBrowserUI/viewModels/DashboardViewModel.swift

```diff
@@ -42,6 +42,12 @@ class DashboardViewModel: ObservableObject {
                 sectionOrder.append(TodaySection(name: section.name, enabled: true))
             }
         }
+        // If item were removed since the last update, remove from the list
+        sectionOrder.forEach { (section) in
+            if !TodaySection.defaultSectionList.contains(section) {
+                sectionOrder.remove(at: sectionOrder.firstIndex(of: section)!)
```

Comment:
you should check for unwrap the index using if let, sectionOrder could not contain it.